.gitignore

```diff
@@ -16,7 +16,7 @@
 .DS_Store
 .env*
 .env
-
+.vs
```

Comment:
can you change this to *.vs to hide all .vs?
<img width="640" alt="Inlab_timeline_rt_by_moblim_·_Pull_Request__155_·_lnccbrown_task-effort" src="https://user-images.githubusercontent.com/29052760/126222187-ce7a389b-4c53-403c-a61d-63adff7fe78e.png">
am getting .vs for conflicting files and that may help